.github/workflows/nix.yml

```diff
@@ -0,0 +1,23 @@
+name: Update nix config
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  nix:
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: cachix/install-nix-action@v27
+        with:
+                github_access_token: ${{ secrets.GITHUB_TOKEN }}
+      - run: |
+          export WORKSPACE="${PWD}"
+          nix run .#upgrade-nix-gohip
+          git config --global user.name 'jgarcia'
+          git config --global user.email 'garciaj.uk@gmail.com'
```

Comment:
probably you be some one realy github-in-the-name user, ie nix@github.com
GH is even now discouraging personal emails in commits log

Reply:
I've changed to something that doesn't really exist, I wasn't sure the other option was a good idea anyway